NOLF2/ClientShellDLL/ClientShellShared/GameClientShell.cpp

```diff
@@ -373,7 +373,7 @@ void CmdFn(int argc, const char **argv)
 	}
 
 	// Send message to server...
-	char *buf = new char(CommandLineLength(argc,argv)+3);
+	char *buf = new char[CommandLineLength(argc,argv)+3];
```

Comment:
if you do array allocation... also do the array delete of buf on [L387]

Reply:
Done.

Reply:
perfect, thanks